include/gatekeeper_ggu.h

```diff
@@ -47,35 +47,72 @@ struct ggu_config {
 	struct mailbox    mailbox;
 };
 
+/* Enumeration of policy decisions the GGU block can process. */
+enum {
+	/* Decline an IPv4 flow. */
+	GGU_DEC_IPV4_DECLINED = 0,
+	/* Decline an IPv6 flow. */
+	GGU_DEC_IPV6_DECLINED,
+	/* Grant an IPv4 flow. */
+	GGU_DEC_IPV4_GRANTED,
+	/* Grant an IPv6 flow. */
+	GGU_DEC_IPV6_GRANTED,
+	__MAX_GGU_DEC
+};
+
+/* On the wire policy decision to be processed by the GGU. */
+struct ggu_decision {
+	/* The policy decision type. */
+	uint16_t type;
```

Comment:
Shouldn't ``uint8_t`` be enough for the type? Maybe we can have one more byte to be reserved for future use.